### PR TITLE
CloudFlare: add the configuration for logpush

### PR DESCRIPTION
**Pull request by squioc**
Add a section about how to forward to SEKOIA.IO with logpush

### CODE REVIEW

File: docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md [hunk 1]
```diff
@@ -25,12 +25,40 @@ To do so, connect to [Cloudflare Console](https://dash.cloudflare.com/) to colle
...
 - `Cloudflare account email`
 
+#### with logpush
 
-You now have all information to configure the Cloudflare module and its `Send events` action to SEKOIA.IO.
+!!! note
+
+    It's the recommended way to get your events from Cloudflare.
+
+Configure a [logpull job](https://developers.cloudflare.com/logs/reference/logpush-api-configuration/) with the following destination:
```
Comment (PierrickV) [inline docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md:34]:
```suggestion
Configure a [Logpush job](https://developers.cloudflare.com/logs/reference/logpush-api-configuration/) with the following destination:
```

File: docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md [hunk 2]
```diff
@@ -25,12 +25,40 @@ To do so, connect to [Cloudflare Console](https://dash.cloudflare.com/) to colle
...
+    "max_upload_bytes": 5000000,
+    "max_upload_records": 1000,
+    "destination_conf": "https://intake.sekoia.io/plain/batch?header_X-SEKOIAIO-INTAKE-KEY=<INTAKE_KEY>"
+}'
+```
+
+
+#### with logpull
 
 You can now enable Cloudflare log retention (see [associated documentation](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/)).
```
Comment (PierrickV) [inline docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md:58]: I didn't get why this section start with this sentence, maybe you deleted something ?

Comment (squioc) [reply docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md:58]: This phrase `You now have all information to configure the Cloudflare module and its `Send events` action to SEKOIA.IO.` was deleted

File: docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md [hunk 3]
```diff
@@ -25,12 +25,40 @@ To do so, connect to [Cloudflare Console](https://dash.cloudflare.com/) to colle
...
```
Comment (squioc) [inline docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md:58]: Here is a rewording suggestion
```suggestion
 To allow the pulling of logs, enable [Cloudflare log retention](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/). 
```

Comment (PierrickV) [reply docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md:58]: Maybe

```suggestion
 To start using Logpull, start by enabling [Cloudflare log retention](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/). 
```

**Review by PierrickV — DISMISSED** [review]

**Review by PierrickV — APPROVED** [review]

**Pull request merged**